python/setup.py.in

```diff
@@ -164,14 +164,7 @@ packages=['paddle',
           'paddle.fluid.contrib.quantize',
           'paddle.fluid.contrib.reader',
           'paddle.fluid.contrib.slim',
-          'paddle.fluid.contrib.slim.core',
-          'paddle.fluid.contrib.slim.graph',
-          'paddle.fluid.contrib.slim.prune',
           'paddle.fluid.contrib.slim.quantization',
-          'paddle.fluid.contrib.slim.quantization.imperative',
```

Comment:
`paddle.fluid.contrib.slim.quantization.imperative`  is added recently, do not delete this.

Reply:
fixed